### ISSUE TITLE
cutadapt in cmd 

### ISSUE THREAD

**Issue by AnaSofia94** [marcelm/cutadapt#334]
Hello, 

i installed cutadapt in cmd
and added its path to windows
but, when i call the package it says, the comand isn't recognized. 

Many thanks

**Comment by AnaSofia94** [comment]
Hello, thank you for your reply
To more specific, i installed cutadapt 1.18 throw the download:
https://pypi.org/project/cutadapt/. then i add the path to my computer, but
when i try to use the program in cmd,
i have this message: 'cutadapt' is not recognized as an internal or
external command.
Im using windows 10.
Thanks


Marcel Martin <notifications@github.com> escreveu no dia quarta, 12/09/2018
à(s) 22:20:

**Comment by marcelm** [comment]
I don’t think you can install Cutadapt like that on Windows. Cutadapt contains parts that need to be compiled and for that, you need a C compiler. Some people managed to get it to work, though, but I don’t know how exactly.

I’m very sorry, but you will need to find someone else to help you with this.

**Comment by AnaSofia94** [comment]
thanks for you help, i already solved the problem installing the program
with anaconda.

Marcel Martin <notifications@github.com> escreveu no dia quinta, 13/09/2018
à(s) 10:16:
